<launch>
  <node pkg="deltarobot_kinematics" exec="kinematics" name="delta_kinematics" >
    <param from="$(find-pkg-share deltarobot_kinematics)/delta_config.yaml"/>
  </node>
</launch>